Readme.md

```diff
@@ -217,3 +229,17 @@ List of user folders:
 Custom textures have to be placed in the user directory under
 `Load/Textures/[GameID]/`. You can find the Game ID by right-clicking a game
 in the ISO list and selecting "ISO Properties".
+
+## Downloading Pull Request (PR) Builds
+
+Windows, macOS and Android PR builds can be downloaded for testing purposes after
+[dolphin-emu-bot](https://github.com/dolphin-emu-bot) builds them.
+
+1. If needed, click "Show all checks" near the bottom of a PR.
+2. Find the appropriate build for your operating system and click "Details"
+3. Click the link that starts with "pr-XXXX-dolphin-latest" to download the PR build, where XXXX is the PR number.
+
+## Dolphin Redirector
```

Comment:
I don't see the point with mentioning this here. It's not technically related to the emulator.